web-marketplace/src/pages/CreditClassDetails/tables/CreditClassDetails.TableTabs.config.tsx

```diff
@@ -11,10 +13,11 @@ export const getCreditClassDetailsTabs = ({
   creditBatches,
   initialPaginationParams,
   onTableChange,
+  _,
 }: CreditClassDetailsTableTabsProps): IconTabProps[] =>
   [
     {
-      label: 'Credit Issuance',
+      label: _(msg`Credit Issuance`),
       content: (
         <CreditBatches
```

Comment:
will `CreditBatches` be handled separately?

Reply:
Yes, I planned to handle components translation separately.

Reply:
ok makes sense